ebpf/state.go

```diff
@@ -267,6 +276,28 @@ func (ns *networkState) mergeConnections(id string, active map[string]*Connectio
 	return conns
 }
 
+func (ns *networkState) updateConnWithStatsAndActiveConn(client *client, key string, active ConnectionStats, closed *ConnectionStats) {
```

Comment:
`updateConnWithStatWithActiveConn`?

Also add a comment on exactly why we need this function over `updateConnWithStats`